azure-quantum/azure/quantum/job/base_job.py

```diff
@@ -208,7 +208,7 @@ def from_storage_uri(
     def upload_input_data(
         container_uri: str,
         input_data: bytes,
-        content_type: str,
+        content_type: str = "application/json",
```

Comment:
i'm not sure if this should be an optional parameter. only the caller knows the right content_type on the blob.

Reply:
I agree, also I suggest to change the type from `str` to `ContentType`

Reply:
This creates a circular dependency b/w base_job and problem code files. Is there another way to do it without importing the ContentType from problem?